src/clj/game/cards-ice.clj

```diff
@@ -825,6 +825,27 @@
                  :cost [:credit 1] :effect (effect (add-prop target :advance-counter 1))}]
     :subroutines [(tag-trace 2)]}
 
+   "Mausolus"
+   {:advanceable :always
+    :subroutines [{:label "Gain 1 (Gain 3)"
```

Comment:
For consistency with other cards, this can be: 
`"Gain 1 [Credits] (Gain 3 [Credits])"`